app/controllers/saved_jobs_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SavedJobsController < ApplicationController
+
+  def create
+    job = Job.find(params[:job_id])
```

Comment:
@NicoleKMarino: we can refactor this:
pull out job = Job.find(params[:job_id]) into a new method, and call this method from create and destroy